plugins/redis.json

```diff
@@ -0,0 +1,19 @@
+{
+    "name": "redis",
+    "version": "0.0.1",
+    "match": "^redis$",
+    "readme": "Running `devbox services start redis` will start redis as a daemon in the background. \n\nYou can manually start Redis in the foreground by running `redis-server $REDIS_CONF --port $REDIS_PORT`. \n\nLogs, pidfile, and data dumps are stored in `.devbox/virtenv/redis`. You can change this by modifying the `dir` directive in `devbox.d/redis/redis.conf`",
```

Comment:
FYI, `devbox info` (also printed after installation) automatically includes services information. Ideally we don't add it twice, but maybe we could make the services section in `info` more readable.

Reply:
I mainly added this comment to emphasize that it's run with the --daemonize flag. 

In the future I think we could show the full start command for the service, since that can provide useful information